jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrReport.java

```diff
@@ -80,4 +101,69 @@ public String getLocation() {
     public void setLocation(String location) {
         this.location = location;
     }
+
+    public void setJdrUuid(String jdrUuid) {
+        this.jdrUuid = jdrUuid;
+    }
+
+    /**
+     * sets JDR UUID using jdr.properties file
+     * if the JDR does not have a UUID, then generate one for them and store it in the jdr.properties file
+     */
+    private void setJdrUuid() {
+        String jbossConfig = System.getProperty(JBOSS_PROPERTY_DIR);
+        // JDR is being ran from command line
+        if(jbossConfig == null) {
+            // if JBoss standalone directory does not exist then go no further
+            if(!new File(DEFAULT_PROPERTY_DIR).exists()) {
+                FileNotFoundException e = new FileNotFoundException("Could not find JDR properties file");
+                e.printStackTrace();
```

Comment:
EAP exceptions need to be i18n, and should be thrown or logged via a logger, not System.err. The i18n class JdrLogger should be used; e.g. JdrLogger.varNull.